android/src/main/java/com/dooboolab/RNIap/RNIapModule.java

```diff
@@ -319,7 +319,12 @@ public void onPurchaseHistoryResponse(BillingResult billingResult, List<Purchase
             for (PurchaseHistoryRecord purchase : purchaseHistoryRecordList) {
               WritableMap item = Arguments.createMap();
               item.putString("productId", purchase.getSku());
-              item.putString("transactionId", purchase.getOrderId());
+              
+              // item.putString("transactionId", purchase.getOrderId());
+              // PurchaseHistoryRecord doesn't have orderId, 
+              // and it seems that getOriginalJson in this case also doesn't return the orderId
+              // is there any way to obtain this?
```

Comment:
Not in `android`'s billing client version 2. Why not using getPurchases instead to acquire this?